content/md/what-is/what-is-a-rule.md

```diff
@@ -14,7 +14,6 @@ For instance, using rules you can:
 - automatically categorise new products based their family
 - copy the value from a text attribute to another text attribute
 - set a default value to an empty field
-- set a text attribute to another text attribute
```

Comment:
It's strange, I have always this line :(

Reply:
@Delphineray do you agree to remove it?